app/controllers/posts_controller.rb

```diff
@@ -6,6 +6,36 @@ def index
     @posts -= [@post]
   end
 
+  def new
+   posts
```

Comment:
Also, _ideally_ @posts should be the bare minimum number of posts you need. That way you don't need to limit them in your view and (more importantly), you're not carrying every post in the system on every request.